src/components/ProductAnnouncement.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+
+type Props = {
+  announcements: { name: string; value: string; displayOrder: number }[]
+}
+
+const ProductAnnouncement = ({ announcements }: Props) => {
+  if (!announcements || announcements.length === 0)
```

Comment:
```suggestion
  if (announcements.length === 0)
```

announcements은 array 타입으로 선언되었기 때문에 falsy한 케이스는 없으므로, 불필요한 조건인것 같아요.

Reply:
그렇군요! 못받아오는 상황을 가정해서 저런식으로 처리했는데 그럴필요가 없었네요. 바로 수정했습니다!